<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IMusic[*]</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rs.qrc">
    <normaloff>:/icons/icons8-музыка-40.png</normaloff>:/icons/icons8-музыка-40.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>1</y>
      <width>551</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEdit"/>
    <addaction name="actionAdd_Artist"/>
    <addaction name="actionAdd_Album"/>
    <addaction name="actionAddTrack"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Programm"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionAdd_Artist"/>
   <addaction name="actionAdd_Album"/>
   <addaction name="actionAddTrack"/>
   <addaction name="actionEdit"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="rs.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="rs.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>save file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAdd_Artist">
   <property name="icon">
    <iconset resource="rs.qrc">
     <normaloff>:/icons/icons8-диджей-40.png</normaloff>:/icons/icons8-диджей-40.png</iconset>
   </property>
   <property name="text">
    <string>Add Artist</string>
   </property>
   <property name="toolTip">
    <string>add new artist</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionAdd_Album">
   <property name="icon">
    <iconset resource="rs.qrc">
     <normaloff>:/icons/icons8-табулатура-40.png</normaloff>:/icons/icons8-табулатура-40.png</iconset>
   </property>
   <property name="text">
    <string>Add Album</string>
   </property>
   <property name="toolTip">
    <string>add new album</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionAddTrack">
   <property name="icon">
    <iconset resource="rs.qrc">
     <normaloff>:/icons/icons8-пластинка-40.png</normaloff>:/icons/icons8-пластинка-40.png</iconset>
   </property>
   <property name="text">
    <string>Add Track</string>
   </property>
   <property name="toolTip">
    <string>add new track</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="rs.qrc">
     <normaloff>:/icons/icons8-стереть-символ-40.png</normaloff>:/icons/icons8-стереть-символ-40.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>delete element</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="rs.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="rs.qrc">
     <normaloff>:/icons/icons8-редактировать-40.png</normaloff>:/icons/icons8-редактировать-40.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="toolTip">
    <string>edit element</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAbout_Programm">
   <property name="icon">
    <iconset resource="rs.qrc">
     <normaloff>:/icons/inf1.png</normaloff>:/icons/inf1.png</iconset>
   </property>
   <property name="text">
    <string>About Programm</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="rs.qrc">
     <normaloff>:/icons/inf2.png</normaloff>:/icons/inf2.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rs.qrc"/>
 </resources>
 <connections/>
</ui>
